btr-api/src/btr_api/resources/submission.py

```diff
@@ -127,26 +127,26 @@ def create_register():
         # save before attempting invoice creation so that we can log the id for ops if there's an error
         submission.save_to_session()
         try:
+            # NOTE: this will be moved out of this api once lear filings are linked
```

Comment:
is there a ticket, ticket number would be good if there is :D

Reply:
I know I tried to something on the board, but we don't even have an epic for it yet (it'll be in next PI). It's on my list though so it'll get done